tests/libtransmission/rename-test.cc

```diff
@@ -212,7 +212,7 @@ TEST_F(RenameTest, singleFilenameTorrent)
     // (while it's renamed: confirm that the .resume file remembers the changes)
     tr_torrentSaveResume(tor);
     sync();
-    loaded = tr_torrentLoadResume(tor, ~0, ctor, nullptr);
+    loaded = tr_torrentLoadResume(tor, ~uint64_t{}, ctor, nullptr);
```

Comment:
Maybe it's better to use Interger Literals? Like `0llu`

Reply:
:+1:   215a3d0ca14e17a72ffdeb7188cce597d71c37fa